### ISSUE TITLE
reset time 

### ISSUE THREAD

**Issue by rizerzero** [weareoutman/clockpicker#16]
hi, i made the input  as read only but when a user want to rest the time to blank or none it's not possible so i suggest a reset button beside the validation button 
thank you

**Comment by cnighswonger** [comment]
I second this request. The ability to reset the time to blank (or some user-defined string) would be fantastic. Thanks for a great widget!

**Comment by vsamiev** [comment]
Hi tried to implement reset function
tested it and it works on my local environment, but not i am not sure 
Please review the code

```
ClockPicker.prototype.resetValue = function(){
    this.input.prop('value', '');
    value = ''
}
```

**Comment by PeterJC91** [comment]
Yes a reset option is the only thing that is missing in my opinion. Other than that is is an awesome widget. Thanks for that. Any idea how to get such a reset function? I would like to be able to reset it
to the placeholder text that is show initially when nothing has been selected yet.

@vsamiev: How can I review your code (where to place it and how to call it).

**Comment by simartn** [comment]
+1 for this!

**Comment by anuraggupta11** [comment]
how to use reset function any example ?
